tests/run-make/rustdoc-output-path/rmake.rs

```diff
@@ -0,0 +1,7 @@
+// Checks that if the output folder doesn't exist, rustdoc will create it.
+
+use run_make_support::{rustdoc, tmp_dir};
+
+fn main() {
+    rustdoc().input("foo.rs").output(tmp_dir().join("foo/bar/doc")).run();
```

Comment:
I know this is a direct translation of the original test, but the original test doesn't actually check if the path exists -- it only checked that `rustdoc` with non-existent output folder returns a success status code.

Reply:
Should I add this check then? Doesn't hurt to check that too I suppose.